vertx-web/src/main/java/io/vertx/ext/web/impl/ParsableMIMEValue.java

```diff
@@ -27,6 +31,25 @@ public String subComponent() {
     return subComponent;
   }
 
+  @Override
+  public String mediaType() {
+    ensureHeaderProcessed();
+    return component + "/" + subComponent;
+  }
+
+  @Override
+  public String mediaTypeWithParams() {
+    ensureHeaderProcessed();
+    StringBuilder sb = new StringBuilder(component + "/" + subComponent);
+    if (!parameters().isEmpty()) {
```

Comment:
This could checked before creating the `StringBuilder`. Then, if there are no params, we can return directly `component + "/" + subComponent`